stunnel/stunnel.ml

```diff
@@ -126,6 +122,10 @@ let getpid ty =
   | Nopid ->
       failwith "No pid!"
 
+type verify = VerifyPeer | CheckHost
+
+type config = {sni: string option; verify: verify; cert_bundle_path: string}
```

Comment:
AFAICT if a caller doesn't want their stunnel connections to be verified, it is simpler to not specify a config at all, than to set `verify=false`. So would the following type definition suffice?

```
type verification_config = { sni : string option ; cert_bundle path: string }
```